tests/conformance/state/state.go

```diff
@@ -801,12 +801,13 @@ func ConformanceTests(t *testing.T, props map[string]string, statestore state.St
 			assertEquals(t, "⏱️", res)
 
 			// Wait for the object to expire and request again
-			time.Sleep(3 * time.Second)
-			res, err = statestore.Get(context.Background(), &state.GetRequest{
-				Key: key + "-ttl",
-			})
-			require.NoError(t, err)
-			assert.Nil(t, res.Data)
+			assert.Eventually(t, func() bool {
```

Comment:
Do we still need this change?

Reply:
Thanks, I'll revert the 70 seconds to 3, but will keep the Eventually block since it speeds up the test